### PR TITLE
STAR-240 adding zebra support - chore(DataGrid)

### PR DESCRIPTION
**Pull request by nahumzs**
### Purpose 🚀
Adding zebra support to the DataGrid

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/STAR-240-DataGrid-zebra

### Screenshots 📸
Screen Shot 2020-05-28 at 3 07 23 PM

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/DataGrid/stories/DataGrid.zebra.stories.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import * as Sbook from "storybook/assets/styles/common.styles";
+import worker from "workerize-loader!./helpers/data.worker"; // eslint-disable-line import/no-webpack-loader-syntax
+import Spinner from "@paprika/spinner";
+import DataGrid from "../src";
+
+export function App() {
+  const [data, setData] = React.useState([]);
+  const [isIdle, setIsIdle] = React.useState(true);
```
Comment (tristanjasper) [inline packages/DataGrid/stories/DataGrid.zebra.stories.js:10]: `isLoading`?  Clearer to use common language here between other components terminology and the `loadData` function.

File: packages/DataGrid/stories/DataGrid.zebra.stories.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      const data = await w.getDataFromWorker(100, 5);
+      setData(() => data);
+      setIsIdle(() => false);
+    }
+
+    loadData();
+  }, []);
+
+  React.useEffect(() => {
+    if (data.length > 0) {
```
Comment (tristanjasper) [inline packages/DataGrid/stories/DataGrid.zebra.stories.js:25]: just `data.length`?

Comment (nahumzs) [reply packages/DataGrid/stories/DataGrid.zebra.stories.js:25]: yes, just checking if data has item, is just an story is running in a controlled env. :)

**Comment by nahumzs** [comment]
I will update all docs at once in my next PR

**Pull request merged**